src/wasm/wasm-type.cpp

```diff
@@ -203,7 +203,9 @@ FeatureSet Type::getFeatures() const {
     switch (t.getSingle()) {
       case Type::v128:
         return FeatureSet::SIMD;
+      case Type::funcref:
       case Type::anyref:
+      case Type::nullref:
         return FeatureSet::ReferenceTypes;
       case Type::exnref:
         return FeatureSet::ExceptionHandling;
```

Comment:
We probably should change this to `FeatureSet::ReferenceTypes | FeatureSet::ExceptionHandling` too

Reply:
Sounds good, will do. Is it possible to use exception handling at all if reference types is not enabled?

Reply:
No. But we currently don't have a way to express feature dependencies in Binaryen, so I'm using `FeatureSet::ReferenceTypes | FeatureSet::ExceptionHandling` wherever I use `FeatureSet::ExceptionHandling`.

I think we once talked about this feature dependencies thing several months ago (which felt like ages ago now), and kind of decided that we take care of that in clang and features section.